tasks/internal_reconcile_report_orphan/adapter_database.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from typing import List, Tuple
+
+from orca_shared.database.shared_db import retry_operational_error, get_user_connection
+from sqlalchemy import text
+from sqlalchemy.future import Engine
+
+from entities import OrphanRecord
+
+
```

Comment:
does this mean if the database/source changes, we only have to update this file?

Reply:
That's the hope. There's a bit of a grey area since the db connection info comes from an AWS secret, but I think I kept that parameter generic enough that it'd work out.